src/pip/_internal/download.py

```diff
@@ -167,6 +167,14 @@ def user_agent():
     # value to make it easier to know that the check has been run.
     data["ci"] = True if looks_like_ci() else None
 
+    provisional = os.environ.get("PIP_USER_AGENT_PROVISIONAL_STRING")
```

Comment:
This should get removed, right?

Reply:
Whoops, thanks. I accidentally left that in fixing the merge conflict while rebasing.